src/app/shared/components/filters-list/filters-list.component.html

```diff
@@ -2,9 +2,9 @@
   <div fxLayout="row" fxLayoutAlign="space-between">
     <button mat-raised-button class="resetFilter" (click)="onFilterReset()">
       <mat-icon class="matFilter">delete_forever</mat-icon>
-      Очистити фільтри
+      Очистити Фільтри
```

Comment:
if Capiyalizing is a general rule for buttons labels you can add global style for buttons to avoid such bugs in future:
`text-transform: capitalize;`